lib/MusicBrainz/Server/Data/Area.pm

```diff
@@ -134,6 +134,15 @@ sub update
     return 1;
 }
 
+sub is_release_country_area {
```

Comment:
I would call this in `can_delete` below too, since `can_delete` is also called by `Edit::Generic::Delete` as a final check, which might matter if someone manages to actually enter such an edit by way of another bug.

Reply:
Probably sensible. Done!